docker/oso-rhel7-zagg-client/monitoring_scripts/cron-send-docker-metrics.py

```diff
@@ -0,0 +1,173 @@
+
+        dds.data_space_available = DockerWatcher.convert_to_size_in_gb( \
+                                self.get_driver_status_attr('Data Space Available'))
+
+        dds.data_space_total = DockerWatcher.convert_to_size_in_gb( \
+                                self.get_driver_status_attr('Data Space Total'))
+
+        dds.metadata_space_used = DockerWatcher.convert_to_size_in_gb( \
+                                self.get_driver_status_attr('Metadata Space Used'))
+
+        dds.metadata_space_available = DockerWatcher.convert_to_size_in_gb( \
+                                self.get_driver_status_attr('Metadata Space Available'))
+
+        dds.metadata_space_total = DockerWatcher.convert_to_size_in_gb( \
+                                self.get_driver_status_attr('Metadata Space Total'))
+
+
+        # Determine if docker is using a loopback device
+        # FIXME: find a better way than allowing this to throw
+        try:
+            self.get_driver_status_attr('Data loop file')
+            dds.is_loopback = True
+        except IndexError:
+            dds.is_loopback = False
+
+        # Work around because loopback lies about it's actual total space
+        if not dds.is_loopback:
+            dds.data_space_total = dds.data_space_used + dds.data_space_available
+            dds.metadata_space_total = dds.metadata_space_used + dds.metadata_space_available
+
+
+        dds.data_space_percent_available = (dds.data_space_available / dds.data_space_total) * 100
+        dds.metadata_space_percent_available = (dds.metadata_space_available / dds.metadata_space_total) * 100
+
+        return dds
+
+if __name__ == "__main__":
+    keys = None
+    exit_code = 0
+    zs = ZaggSender()
+    try:
+        cli = AutoVersionClient(base_url='unix://var/run/docker.sock')
+        dw = DockerWatcher(cli)
+        dw_dds = dw.get_disk_usage()
+
+        keys = {
+            'docker.storage.data.space.used': dw_dds.data_space_used,
+            'docker.storage.data.space.available': dw_dds.data_space_available,
+            'docker.storage.data.space.percent_available': dw_dds.data_space_percent_available,
+            'docker.storage.data.space.total': dw_dds.data_space_total,
+
+            'docker.storage.metadata.space.used': dw_dds.metadata_space_used,
+            'docker.storage.metadata.space.available': dw_dds.metadata_space_available,
+            'docker.storage.metadata.space.percent_available': dw_dds.metadata_space_percent_available,
+            'docker.storage.metadata.space.total': dw_dds.metadata_space_total,
+
+            'docker.storage.is_loopback': int(dw_dds.is_loopback),
+            'docker.ping': 1, # Docker is up
+        }
+    except DockerException as ex:
+        print "\nERROR talking to docker: %s\n" % ex.message
+        keys = {
+            'docker.ping': 0,  # Docker is down
+        }
+        exit_code = 10
+
+    zs.add_zabbix_keys(keys)
+
+    print "Sending these metrics:"
```

Comment:
I debated this, and for this quick hack I determined yes. For a real version, I'll probably add -v :)